update-center-source.properties

```
@@ -1,11 +1,19 @@
 plugins=abap,aemrules,ansible,sonarapex,apigee,authaad,authbitbucket,authgithub,authgitlab,authgoogleoauth,authoidc,authsaml,branch,buildbreaker,checkstyle,citymodel,cloudformation,clover,cnesreport,cobertura,cobol,codehawk,communitybsl,cpp,crowd,csharp,cssfamily,dependencycheck,detekt,developer,devon4j,findbugs,flex,fxcop,go,governance,groovy,ha,hadolint,icode,jacoco,java,javai18n,javascript,jdepend,jqassistant,kotlin,ldap,license,lits,l10nde,l10nfr,l10nel,l10nes,l10nit,l10nja,l10nko,l10npt,l10nru,l10nzh,l10nzhtw,mulesoft,mutationanalysis,mybatis,pam,php,pli,plsql,pmd,python,quboo,shellcheck,rci,rpg,ruby,sonarscala,scmcvs,scmgit,scmjazzrtc,scmtfvc,scmmercurial,scmsvn,smells,softvis3d,sonargraphintegration,swift,timeline,tsql,typescript,webdriver,vb,vbnet,web,xanitizer,xml,yaml
 scanners=scannercli,scannermsbuild,scannerant,scannermaven,scannergradle,scannerazure,scannerjenkins
 
-devVersion=8.9-SNAPSHOT
-publicVersions=7.9,7.9.1,7.9.2,7.9.3,7.9.4,7.9.5,7.9.6,8.0,8.1,8.2,8.3,8.3.1,8.4,8.4.1,8.4.2,8.5,8.5.1,8.6,8.6.1,8.7,8.7.1,8.8
+devVersion=9.0-SNAPSHOT
+publicVersions=7.9,7.9.1,7.9.2,7.9.3,7.9.4,7.9.5,7.9.6,8.0,8.1,8.2,8.3,8.3.1,8.4,8.4.1,8.4.2,8.5,8.5.1,8.6,8.6.1,8.7,8.7.1,8.8,8.9
 privateVersions=4.2,4.3,4.3.1,4.3.2,4.3.3,4.4,4.4.1,4.5,4.5.1,4.5.2,4.5.4,4.5.5,4.5.6,4.5.7,5.0,5.0.1,5.1,5.1.1,5.1.2,5.2,5.3,5.4,5.5,5.6,5.6.1,5.6.2,5.6.3,5.6.4,5.6.5,5.6.6,5.6.7,6.0,6.1,6.2,6.3,6.3.1,6.4,6.5,6.6,6.7,6.7.1,6.7.2,6.7.3,6.7.4,6.7.5,6.7.6,6.7.7,7.0,7.1,7.2,7.2.1,7.3,7.4,7.5,7.6,7.7,7.8
 
-ltsVersion=7.9.6
+ltsVersion=8.9
+
+8.9.description=Long-Term Support version - Many bug fixes and small improvements
```

```suggestion
8.9.description=Long-Term Support version - Hardening for long-term reliability, plus minor bug fixes and small improvements
```

---

Does it work for you?
```suggestion
8.9.description=Long-Term Support version - Hardening for long-term reliability, bug fixes and small improvements
```

---

Sure